tokio/src/runtime/blocking/pool.rs

```diff
@@ -338,6 +338,10 @@ impl Spawner {
         let _ = name;
 
         let (task, handle) = task::unowned(fut, NoopSchedule, id);
+
+        #[cfg(feature = "test-util")]
+        crate::time::inhibit_auto_advance();
```

Comment:
Just realized this leaves the counter incremented forever if the task panics.

I could fix this by creating a little value that does `allow_auto_advance()` when dropped. Then thread that through everything so it gets dropped when either the task is done, or something panics.

Not going to fix it _right_ now because I'm not sure y'all even want any of my crummy code, :D but let me know and I'll be happy to code that up.

Reply:
This is definitely something that would need to happen in `Drop` for it to be robust. We also need to consider the case where spawning a new thread in the `spawn_blocking` threadpool fails.

Reply:
OK, fixed now.